tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -61,6 +61,15 @@ Define {
 	DEEP_BUFFER_PIPELINE_SRC	'mixin.15.1'
 	DEEP_BUFFER_PIPELINE_SINK	'mixout.2.1'
 	DEEP_BUFFER_PCM_NAME		'Deepbuffer Jack Out'
+	HEADSET_CODEC_NAME		'SSP0-Codec'
+	HEADSET_PLAYBACK_PCM_CAPS	'HS Playback'
+	HEADSET_CAPTURE_PCM_CAPS	'HS Capture'
```

Comment:
if this the PCM name or the stream name? I don't get what the 'CAPS' stand for or relate to.

Reply:
This is the PCM capabilities name but maybe we can rename it to HEADSET_CAPTURE_PCM_STREAM_NAME/HEADSET_PLAYBACK_PCM_STREAM_NAME

Reply:
Done